framework/src/mesh/MooseMesh.C

```diff
@@ -115,14 +115,22 @@ validParams<MooseMesh>()
                              "Specifies the sort direction if using the centroid partitioner. "
                              "Available options: x, y, z, radial");
 
-  MooseEnum patch_update_strategy("never always auto", "never");
-  params.addParam<MooseEnum>("patch_update_strategy",
-                             patch_update_strategy,
-                             "How often to update the geometric search 'patch'.  The default is to "
-                             "never update it (which is the most efficient but could be a problem "
-                             "with lots of relative motion).  'always' will update the patch every "
-                             "timestep which might be time consuming.  'auto' will attempt to "
-                             "determine when the patch size needs to be updated automatically.");
+  MooseEnum patch_update_strategy("never always auto nonlinear_iter", "never");
+  params.addParam<MooseEnum>(
+      "patch_update_strategy",
+      patch_update_strategy,
```

Comment:
The default is 'never', so change the text accordingly.

Reply:
I have changed the text to match the default 'never' option

Reply:
I still don't have a better name for the patch update strategy than 'nonlinear_iter'. Do you have any suggestions?

Reply:
Actually, how about just 'iteration'?

Reply:
I have changed the name for the patch update strategy form 'nonlinear_iter' to 'iteration'